projects/igniteui-angular/src/lib/grids/grid/row-drag.directive.spec.ts

```diff
@@ -899,6 +900,46 @@ describe('IgxGrid - Row Drag Tests', () => {
             verifyRowDragEndEvent(dragGrid, rowToDrag, rowDragDirective, false, 3);
         }));
     });
+    describe('Tree Grid Deleting Tests', () => {
+        let dragGrid: IgxTreeGridComponent;
+        let dragRows: DebugElement[];
+        beforeEach(async(() => {
+            fixture = TestBed.createComponent(IgxTreeGridDeleteRowTestComponent);
```

Comment:
You can probably reuse the `IgxTreeGridTestComponent` here, just don't use hardcoded drop handlers.